pages/advanced-algorithms/available-algorithms/leiden_community_detection.mdx

```diff
@@ -0,0 +1,160 @@
+nodes.
+
+This module employs the [Leiden
+algorithm](https://en.wikipedia.org/wiki/Leiden_algorithm) for community detection
+based on paper [*From Louvain to Leiden: guaranteeing well-connected communities*](https://arxiv.org/abs/1810.08473). 
+The Leiden algorithm is a hierarchical clustering algorithm, that recursively merges communities into single nodes by greedily optimizing the modularity and the process repeats in the condensed graph.
+It enhances the Louvain algorithm by addressing its limitations, particularly in situations where some identified communities are not well-connected. 
+This improvement is made by periodically subdividing communities into smaller, well-connected groups.
+With an $\mathcal{O}(Lm)$ runtime for $m$ edges and $L$ number of iterations, it suits large graphs.
+
+<Cards>
+  <Card
+    icon={<GitHub />}
+    title="Source code"
+    href="https://github.com/memgraph/mage/blob/main/cpp/leiden_community_detection_module/leiden_community_detection_module.cpp"
+  />
+</Cards>
+
+| Trait                    | Value                                                                                                       |
+| ------------------------ | --------------------- |
+| **Module type**          | algorithm             |
+| **Implementation**       | C++                   |
+| **Graph direction**      | undirected            |
+| **Relationship weights** | weighted / unweighted |
+| **Parallelism**          | parallel              |
+
+## Procedures
+
+<Callout type="info">
+You can execute this algorithm on [graph projections, subgraphs or portions of the graph](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph).
+</Callout>
+
+### `get()`
+
+Computes graph communities using the Leiden algorithm.
+
+{<h4> Input: </h4>}
+
+- `subgraph: Graph` (**OPTIONAL**) ➡ A specific subgraph, which is an [object of type Graph](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph) returned by the `project()` function, on which the algorithm is run.
+- `weight: string (default=null)` ➡ Specifies the default relationship weight. If not set, 
+  the algorithm uses the `weight` relationship attribute when present and otherwise 
+  treats the graph as unweighted.
+- `gamma: double (default=1.0)` ➡ Resolution parameter used when computing the modularity. Internally the value is divided by the number of relationships for an unweighted graph, or the sum of weights of all relationships otherwise.
+- `theta: double (default=0.01)` ➡ Controls the randomness while breaking a community into smaller ones.
+- `resolution_parameter: double (default=0.01)` ➡  Minimum change in modularity that must be achieved when merging nodes within the same community.
+- `max_iterations: int (default=inf)` ➡ Maximum number of iterations the algorithm will perform. If set to infinity, the algorithm will run until convergence is reached.
+
+{<h4> Output: </h4>}
+
+- `node: Vertex` ➡ Graph node.
```

Comment:
Can we give this a more descriptive label? What is special about this single node that is returned? Is it some kind of centroid for the community?